scripts/runtestsuite/runtestsuite.py

```diff
@@ -238,15 +328,31 @@ def report_detailed_failure(self, scene, reference_filepath, output_filepath, lo
     def __write_header(self, args):
         script_path = os.path.realpath(__file__)
 
+        git_hash, git_title = git_information(self.template_directory)
+
         self.file.write(self.__render(self.header_template,
                                       {'test-date': CURRENT_TIME,
                                        'python-version': utils.get_python_version(),
                                        'script-path': script_path,
                                        'script-version': VERSION,
                                        'appleseed-binary-path': args.tool_path,
                                        'max-abs-diff-allowed': VALUE_THRESHOLD,
-                                       'max-diff-comps-count-allowed': MAX_DIFFERING_COMPONENTS}))
+                                       'max-diff-comps-count-allowed': MAX_DIFFERING_COMPONENTS,
+                                       'git-hash': git_hash,
+                                       'git-title': git_title}))
         self.file.flush()
+    
+    def __write_stats(self, results):
+        total_time_text = results.total_time()
+        success_rate_text = "{0}%".format(results.success_rate())
+        failures_text = "{0} out of {1} test scene(s)".format(
+            results.failure_count(), 
+            results.total_count())
```

Comment:
```suggestion
            results.total_test_count())
```

`total_count` doesn't really mean anything on its own right ?

Reply:
Good point